index.html

```diff
@@ -152,7 +178,9 @@ <h2 class="title-card">About Myself</h2>
            <li><a href=""><img src="images/social-media-icon/github.png" alt="github"></a></li>
            <li><a href=""><img src="images/social-media-icon/v.png" alt="v"></a></li>
         </ul>
-        <button class="btn"><a href="">Get my resume</a></button>
+        <button class="btn btn-1"><a href="">Get my resume</a></button>
+    </div>
```

Comment:
- [ ] For the `About Myself` section, Please notice the left one is expected to take 40% and the right one the remaining 60% of the page's width. In your case, I can see both use 50% of the width and this is against the provided design, Please amend this change to the project. 
- [ ] Also, I can see your design overflows for screen size from `768px` to `1400px`, and I have to scroll the overflow to see the whole content. This is not a good practice when it comes to the UI design guideline. Please make sure your design is responsive and stays inside the view port at all time.

### Your design 
![image](https://github.com/jerymey/Portfolio-setup/assets/69077061/51c406c9-5ff4-4bf9-9431-eb90b4989385)

### Figma design 
![image](https://github.com/jerymey/Portfolio-setup/assets/69077061/08e0e2cc-7f32-43be-ab69-4fb92a52f23f)